src/main/java/com/ericsson/eiffel/remrem/semantics/constants/RemremSemanticsConstants.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * 
+ */
+package com.ericsson.eiffel.remrem.semantics.constants;
+
+/**
+ * @author erjamni
+ *
+ */
+public interface RemremSemanticsConstants {
```

Comment:
You should normally not add constants to an interface. https://dev.eclipse.org/sonar/rules/show/squid:S1214?modal=true&layout=false